src/Microsoft.DotNet.Build.Tasks/PackageFiles/tool-runtime/project.csproj

```diff
@@ -4,8 +4,8 @@
     <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>
     <EnableDefaultItems>false</EnableDefaultItems>
     <RuntimeFrameworkVersion Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">2.0.0-beta-001776-00</RuntimeFrameworkVersion>
-    <PackageTargetFallback Condition=" '$(TargetFramework)' == 'netcoreapp2.0' ">$(PackageTargetFallback);dnxcore50;portable-net45+win8</PackageTargetFallback>
     <OutputType>Exe</OutputType>
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
```

Comment:
Do we want to disable this warning or actually fix the package closure?

Reply:
I think we've discussed this before, right?

`<PackageReference Include="Microsoft.Cci" Version="4.0.0-rc4-24217-00" />`

We need to build a new `Microsoft.Cci` package, which no one is committing to do.

Reply:
We fixed it on buildtools master by only depending on stable versions of all packages, but I thought that it was probably a risky change to change dependencies on a release/2.0 branch, which is why I just suppressed the error since this was a problem already, it was just treated as a warning before.